diff --git a/project/selfridges-group-website/sites/all/modules/contrib/colorbox_node/colorbox_node.js b/project/selfridges-group-website/sites/all/modules/contrib/colorbox_node/colorbox_node.js
index d4e248f..3ceed5d 100644
--- a/project/selfridges-group-website/sites/all/modules/contrib/colorbox_node/colorbox_node.js
+++ b/project/selfridges-group-website/sites/all/modules/contrib/colorbox_node/colorbox_node.js
@@ -185,18 +185,20 @@
                 });
             }
 
-            function getIndex(increment) {
-                var max = $related.length;
-                var newIndex = (idx + increment) % max;
-                return (newIndex < 0) ? max + newIndex : newIndex;
-            }
+        }
+        
+        // SELFGW-82: https://drupal.org/node/2218251, comment #4
+        function getIndex(increment) {
+          var max = $related.length;
+          var newIndex = (idx + increment) % max;
+          return (newIndex < 0) ? max + newIndex : newIndex;
+        };
 
-            // Find a colorbox link by href in an array
-            function findHref(items, href){
-                return $.grep(items, function(n, i){
-                    return n.href == href;
-                });
-            };
+        // Find a colorbox link by href in an array
+        function findHref(items, href) {
+          return $.grep(items, function(n, i){
+              return n.href == href;
+          });
         }
     }
 
